test/run.d

```diff
@@ -166,9 +167,15 @@ void ensureToolsExists(const TestTool[] tools ...)
             ] ~ tool.extraArgs;
 
             writefln("Executing: %-(%s %)", command);
-            spawnProcess(command).wait;
+            if (spawnProcess(command).wait)
+                failCount++;
         }
     }
+    if (failCount)
+    {
+        stderr.writefln("failed to build %s test tool(s)", failCount);
```

Comment:
It would be better to report the names of the tools that failed than the count.

Reply:
That information should already be printed, we actually would probably be fine not printing anything here.  What do you think?

Reply:
If the information is already printed I'm fine with this. Perhaps even unnecessary to print the count.

Reply:
ok, no longer printing count, and making sure to print which tool(s) failed